crowdsourcing/viewsets/project.py

```diff
@@ -89,9 +89,9 @@ class ModuleViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         queryset = Module.objects.all()
-        requesterid=self.request.query_params.get('requesterid',None)
```

Comment:
ah ok, then the lookup_field is id, use get_object it will automatically give you the correct requester